FEXCore/Source/Interface/Core/JIT/JITClass.h

```diff
@@ -171,6 +171,11 @@ class Arm64JITCore final : public CPUBackend, public Arm64Emitter {
     return ConvertSize(Op);
   }
 
+  [[nodiscard]]
+  ARMEmitter::Size ConvertSize(const IR::OpSize Size) {
```

Comment:
Seems like all of these ConvertSize members can be static, but I can just move those over in a follow-up